docs/core/equations.qmd

```diff
@@ -28,35 +28,35 @@ relationships.
 The precipitation term is given by:
 
 $$
-Q_P = P A(S) 
+Q_P = P \cdot A(S).
 $$
 
-Where $P$ is the precipitation rate and $A$ is the wetted area. $A$ is a
-function of storage: as the volume of water changes, the area of the free water
+Here $P$ is the precipitation rate and $A$ is the wetted area. $A$ is a
+function of the storage $S$: as the volume of water changes, the area of the free water
 surface may change as well, depending on the slopes of the surface waters. 
 
 ## Evaporation
 
 The evaporation term is given by:
 
 $$
-Q_E = E_{pot} \cdot A(S) \cdot (0.5 \cdot \tanh(S - 50.0) / 10.0) + 0.5
+Q_E = E_\text{pot} \cdot A(S) \cdot \frac{1}{2}\left( \tanh\left(\frac{S-50}{10}\right) +1\right)
 $$
 
-Where $E_{pot}$ is the potential evaporation rate and $A$ is the wetted area.
-The $(0.5 \cdot \tanh(S - 50.0) / 10.0) + 0.5$ term ensures no evaporation occurs
-when $S <= 0$ and is a sigmoid function which provides a smooth gradient as $S
+Here $E_\text{pot}$ is the potential evaporation rate and $A$ is the wetted area.
+The $\frac{1}{2}\left( \tanh\left(\frac{S-50}{10}\right) +1\right)$ term ensures almost no evaporation occurs
+when $S \le 0$ and is a sigmoid function which provides a smooth gradient as $S
```

Comment:
What does `\le` change?

Reply:
`\le` contracts `<=`  into a single symbol, it stands for lesser than or equal to (it is equivalent to `\leq`).